vizro-core/examples/features/app.py

```diff
@@ -365,19 +365,26 @@
         * RadioItems (**categorical** single option selector only)
         * RangeSlider (**numerical** multi option selector only)
         * Slider (**numerical** single option selector only)
+        * DatePicker(**temporal** multi and single option selector)
 
         """
         ),
         vm.Table(
-            id="table-gapminder", figure=dash_data_table(data_frame=gapminder, page_size=10), title="Gapminder Data"
+            id="table-gapminder",
+            figure=dash_data_table(data_frame=gapminder_2007, page_size=10),
+            title="Gapminder Data",
         ),
         vm.Table(id="table-tips", figure=dash_data_table(data_frame=tips, page_size=10), title="Tips Data"),
     ],
     controls=[
         vm.Filter(
             targets=["table-gapminder"],
-            column="year",
-            selector=vm.RangeSlider(title="Range Slider (Gapminder - year)", step=1, marks=None),
+            column="lifeExp",
+            selector=vm.RangeSlider(
+                title="Range Slider (Gapminder - lifeExp)",
+                step=1,
+                marks=None,
```

Comment:
Did you set the same for the yaml example? Otherwise remove or bring over to the yaml example